### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rubinovitz**
“Kill your darlings, kill your darlings, even when it breaks your egocentric little scribbler’s heart, kill your darlings.”

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # <img src="https://raw.githubusercontent.com/HackathonHackers/code-of-conduct/master/ico.png" alt="Ico" height="32" valign="middle" /> code-of-conduct
 
-Welcome to [Hackathon Hackers](http://hh.gd)! This group is of, by, and for people who love hackathons—whether you attend hackathons, run them, critique them and the industrial-capitalist system that manifests itself around them, or just love to hack, this group is for you. We're glad you're here.
```
Comment (folz) [inline README.md:3]: :'(

Comment (rubinovitz) [inline README.md:3]: What if u don't love hackathons?

Comment (hallie) [inline README.md:3]: Then wai hackathon hack. Wai not just hack hack. Unhackathon Unhackers.

Comment (rubinovitz) [inline README.md:3]: I think it's unrealistic to think we all love hackathons all the time.
HH Skeptics

**Comment by hellyeah** [comment]
I like this version a lot. Really brings the most important parts of the Code to the forefront. Would love to move forward with this branch and contribute to it.

**Pull request merged**